src/main/frontend/pipeline-graph-view/_cards.scss

```diff
@@ -20,60 +20,19 @@
   }
 
   &__item {
-    position: relative;
-    border-radius: 15px;
-    padding: 20px;
-    background: var(--card-background);
-    overflow: auto;
```

Comment:
likely this change causes graph overflow:

<img width="1060" alt="image" src="https://github.com/user-attachments/assets/8ffe4812-28a4-466e-b077-22e7b12c6d33">

```
pipeline {
    agent any
    stages {
        stage('Non-Parallel Stage') {
            steps {
                echo 'This stage will be executed first.'
            }
        }
        stage('Parallel Stage') {
            failFast true
            parallel {
                stage('Branch A') {
                    steps {
                        echo "On Branch A"
                    }
                }
                stage('Branch B') {
                    steps {
                        echo "On Branch B"
                    }
                }
                stage('Branch C') {
                    stages {
                        stage('Nested 1') {
                            steps {
                                echo "In stage Nested 1 within Branch C"
                            }
                        }
                        stage('Nested 2') {
                            steps {
                                echo "In stage Nested 2 within Branch C"
                            }
                        }
                    }
                }
            }
        }
        stage('Skipped stage') {
            when {
                branch 'master'
            }
            steps {
                echo 'This stage will be skipped.'
            }
        }
        stage('Parallel Stage 2') {
            failFast true
            parallel {
                stage('Branch A') {
                    steps {
                        echo "On Branch A"
                    }
                }
                stage('Branch B') {
                    steps {
                        echo "On Branch B"
                    }
                }
                stage('Branch C') {
                    stages {
                        stage('Nested 1B') {
                            steps {
                                echo "In stage Nested 1B within Branch C"
                            }
                        }
                        stage('Nested 2B') {
                            steps {
                                echo "In stage Nested 2B within Branch C"
                            }
                        }
                    }
                }
            }
        }
    }
}
```

Reply:
> likely this change causes graph overflow:
> 
> <img alt="image" width="1060" src="https://private-user-images.githubusercontent.com/21194782/364147497-8ffe4812-28a4-466e-b077-22e7b12c6d33.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjUzOTkwMzksIm5iZiI6MTcyNTM5ODczOSwicGF0aCI6Ii8yMTE5NDc4Mi8zNjQxNDc0OTctOGZmZTQ4MTItMjhhNC00NjZlLWIwNzctMjJlN2IxMmM2ZDMzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTAzVDIxMjUzOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTVlMWQxNjM2NjZiZTllNDRiYzEyZDA2ZjA0MWIxMzAxYzFjNGU0MTk3YzdmOWY3YjMwZDAwZDQ5NzljNTEzZTImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.4HN4NKvSIyVrNobUZzrJTSZ2WEHXNYr1IWskYMZVSnE">
> ```
> pipeline {
>     agent any
>     stages {
>         stage('Non-Parallel Stage') {
>             steps {
>                 echo 'This stage will be executed first.'
>             }
>         }
>         stage('Parallel Stage') {
>             failFast true
>             parallel {
>                 stage('Branch A') {
>                     steps {
>                         echo "On Branch A"
>                     }
>                 }
>                 stage('Branch B') {
>                     steps {
>                         echo "On Branch B"
>                     }
>                 }
>                 stage('Branch C') {
>                     stages {
>                         stage('Nested 1') {
>                             steps {
>                                 echo "In stage Nested 1 within Branch C"
>                             }
>                         }
>                         stage('Nested 2') {
>                             steps {
>                                 echo "In stage Nested 2 within Branch C"
>                             }
>                         }
>                     }
>                 }
>             }
>         }
>         stage('Skipped stage') {
>             when {
>                 branch 'master'
>             }
>             steps {
>                 echo 'This stage will be skipped.'
>             }
>         }
>         stage('Parallel Stage 2') {
>             failFast true
>             parallel {
>                 stage('Branch A') {
>                     steps {
>                         echo "On Branch A"
>                     }
>                 }
>                 stage('Branch B') {
>                     steps {
>                         echo "On Branch B"
>                     }
>                 }
>                 stage('Branch C') {
>                     stages {
>                         stage('Nested 1B') {
>                             steps {
>                                 echo "In stage Nested 1B within Branch C"
>                             }
>                         }
>                         stage('Nested 2B') {
>                             steps {
>                                 echo "In stage Nested 2B within Branch C"
>                             }
>                         }
>                     }
>                 }
>             }
>         }
>     }
> }
> ```

Good spot, should be sorted now